deb-packaging/deb/control/control

```diff
@@ -19,6 +19,6 @@ Version: [[version]]
 Section: misc
 Priority: optional
 Architecture: all
-Depends: default-jre-headless (>= 1.7)
```

Comment:
I'm using `java8-runtime` because Karaf requires java 1.8. Using `default-jre-headless` could potentially install and use java 1.7 in older system which would be problematic when launching the service